config/locales/instructions/en.yml

```diff
@@ -21,13 +21,13 @@ en:
         iv: Good
         v: Great!
     registration:
-      already_have_account: Already have account? %{link}
+      already_have_account: Already have an account? %{link}
       email: Pick an address to use for government communications.
     2fa:
       resend: Didn’t receive a code? %{link}
-      confirm_code: We sent the code to %{number}.
+      confirm_code: We sent it to %{number}.
       wrong_number: Entered the wrong phone number? %{link}
       totp_intro_html: >
-        Please enter the code from your authenticator app.
+        Enter the code from your authenticator app.
```

Comment:
Removed all the "pleases" as per http://www.ihearttechnicalwriting.com/write-error-messages/. Which maybe @jeanninehunter will disagree with 😨